tests/acceptance/package.json

```diff
@@ -26,8 +26,8 @@
   },
   "dependencies": {
     "@currents/playwright": "^1.5.11",
-    "@playwright/test": "1.49.1",
-    "@shopware-ag/acceptance-test-suite": "^10.2.3",
+    "@playwright/test": "^1.45.3",
```

Comment:
Is there reason we had this pinned? playwright gets required by the ATS package itself, so you've to manually find the correct version when you update. It's really annoying to fix, so I've opened up the constraint

Reply:
I change it here, because when we are want to use the ATS within platform we have to make sure that the playwright version in the ATS is the same as in platform. If not you get failures during the run. So to avoid mismatch I pinned it. So just for safety reasons.

Reply:
I'd remove `"@playwright/test": "1.49.1",` too. Even better.

Reply:
I've removed it, seems to work fine